pkg/controller/operator/seed/reconciler.go

```diff
@@ -93,6 +93,13 @@ func (r *Reconciler) reconcile(log *zap.SugaredLogger, seedName string) error {
 		return nil
 	}
 
+	// to allow a step-by-step migration of seed clusters, it's possible to
+	// disable the operator's reconciling logic for seeds
+	if seed.Annotations[common.SkipReconcilingAnnotation] != "" {
```

Comment:
I would rather do a check on the presence of the annotation:
```
if _, ok := seed.Annotations[common.SkipReconcilingAnnotation]; ok {
```

Reply:
Well _technically_ in this case both variants are identical.

Reply:
right

Reply:
nope sorry, I was confused. They are not identical. the zero value for strings is "", we cannot distinguish between non present entry and empty string entry without the `ok` value

Reply:
Well good thing I changed the condition.